tests-beta/android/MASVS-RESILIENCE/MASTG-TEST-0x39-1.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Debuggable Flag Not Disabled in the AndroidManifest
+platform: android
+id: MASTG-TEST-0x39-1
+type: [static]
+weakness: MASWE-0067
+---
+
+## Overview
+
+This test case checks if the application has the debuggable flag set in the AndroidManifest.xml. If this flag is set, an attacker can attach a debugger, and e.g., read and modify internals of the application.
+
+!!! note Other ways to debug the application
+    Not enabling debugging in the AndroidManifest.xml does fully prevent all possibilities to debug the app. See @MASWE-0101 for more details on how to prevent debugging.
```

Comment:
If we know about other ways to debug the app we should shortly note them here.

Do you mean something like the attacker can set this attribute (which would invalidate the signature) and repackage & re-sign and be able to debug the app. I think this is something we could _note_ here.

Reply:
I meant https://github.com/OWASP/owasp-mastg/pull/3042#discussion_r1833094787 here as well.

The resigning is a good point, not sure we want to open this topic here.